config/software/designs.json

```diff
@@ -214,6 +214,13 @@
       "difficulty": 1,
       "tags": ["tops", "coats", "historical"]
     },
+    "otis": {
+      "description": "A FreeSewing pattern that needs a description",
+      "code": "Coder name",
```

Comment:
You might want to update this with a description and your own name. You can check some of your other designs in this file for inspiration :)